calyx/README.md

```diff
@@ -0,0 +1,31 @@
+## Structure
+
+- [src](src)
+  - [main.rs](src/main.rs)
+  - [cmdline.rs](src/cmdline.rs) (parsing the input)
+  - [errors.rs](src/errors.rs) (error message generator)
+  - [utils.rs](src/main.rs) 
+  - [lang/](src/lang/) (building AST from futil inputs)
+  - [passes/](src/passes/) (optimisation passes)
+  - [backend/](src/backend/) (Verilog file generation)
+
+- [examples](examples/) (example files)
+
+- [primitives/](primitives) (libraries)
+
+  
+
+## Makefile
```

Comment:
It is worth mentioning that this is the Makefile under `calyx`. There is also a Makefile in the repo root that has the same targets.